client/components/App.jsx

```diff
@@ -35,18 +35,23 @@ class App extends React.Component {
           rating = {this.state.rating}
           numRatings = {this.state.numRatings}
         />
+        <br></br>
         <Subhead
           condition = {this.state.condition}
           sold = {this.state.numSold}
           availability = {this.state.availability}
         />
+        <br></br>
         <Price
           price = {this.state.price}
         />
+        <br></br>
         <Logistics
           shipping = {this.state.shipping}
         />
+        <br></br>
         <Guarentee />
+        <br></br>
```

Comment:
Love the breaks here I'm sure it adds wonderful spacing to your component. Very clever of you to make use of the br tag here, I respect that a lot. Another one, another one, ano-